apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -39,41 +170,130 @@
     content: ' ';
     display: block;
     visibility: hidden;
-    background-color: red;
+    background-color: #D32E6A;
```

Comment:
Color `#D32E6A` should be written as `#d32e6a`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexnotation)